Scoop/Controller/Home/HomeViewController.swift

```diff
@@ -239,11 +249,18 @@ extension HomeViewController: UITableViewDataSource {
         
         switch TableSection.allCases[indexPath.section] {
         case .activeTrips:
-            cell.configure(ride: activeRides[indexPath.row])
+            let shortenedRide = activeRides[indexPath.row]
+            if let driver = shortenedRide.driver {
+                let fullRide = Ride(id: 0, creator: BaseUser(id: driver.id, netid: driver.netid, firstName: driver.firstName, lastName: driver.lastName, prompts: [], rides: []), maxTravelers: shortenedRide.maxTravelers, minTravelers: shortenedRide.minTravelers, departureDatetime: shortenedRide.departureDatetime, description: shortenedRide.description, isFlexible: true, path: shortenedRide.path, type: shortenedRide.type)
```

Comment:
You use this multiple times, make it into a constant as a private class property